addons/pos_customer_display/__manifest__.py

```diff
@@ -0,0 +1,26 @@
+{
+    'name': 'PoS Customer Display',
+    'version': '1.0.0',
+    'category': 'Sales/Point of Sale',
+    'summary': 'Display customer\'s order',
+    'depends': ['point_of_sale'],
+    'installable': True,
+    'auto_install': True,
```

Comment:
Why is it auto install?
Why is this feature in a separated module? Why not include it built-in into `point_of_sale`?